base/libgit2/callbacks.jl

```diff
@@ -213,56 +219,41 @@ function credentials_callback(libgit2credptr::Ptr{Ptr{Void}}, url_ptr::Cstring,
     # get credentials object from payload pointer
     creds = nothing
     creds_are_temp = true
```

Comment:
no longer occurs anywhere else in the file? this had been triggering a `securezero!(creds)`, is that still needed?